napari/_qt/widgets/qt_viewer_buttons.py

```diff
@@ -141,8 +142,13 @@ def __init__(self, viewer: 'ViewerModel') -> None:
         rdb.customContextMenuRequested.connect(self._open_roll_popup)
 
         self.transposeDimsButton = QtViewerPushButton(
-            'transpose', action='napari:transpose_axes'
+            'transpose',
+            trans._(
+                'Transpose order of last two visible axes.\nAlt/option-click to rotate visible axes)'
```

Comment:
```suggestion
                'Transpose order of last two visible axes.\n(Alt/option-click to rotate visible axes)'
```

Missing a bracket here! Also is there any way we can use the OS symbols in this tooltip...?